### PR TITLE
Refactor mortar activation

### PR DESCRIPTION
**Pull request by VulturARG**

### CODE REVIEW

File: core/scripts/flares/activate_mortar.sqf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		private _enableFlareEnhance = getMissionConfigValue ["ACTIVAR_BENGALAS_MEJORADAS",  0] == 1; 
+		if (_enableFlareEnhance) then {
+			deleteVehicle (_this select 6);
+		};
+	}
+];
+
+/*******************************************************************************
+                          Realizado por MIV
+*******************************************************************************/
```
Comment (IgnacioKase) [inline core/scripts/flares/activate_mortar.sqf]: add a new line at the EOF!

File: core/scripts/flares/activate_mortar.sqf [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+} forEach [1,2,3,4];
+
+_mortar addEventHandler [
+	"Fired",{
+		private _file = "core\scripts\flares\mortar_flare_enhance.sqf";
+		[[getPosASL (_this select 0), velocity (_this select 6), _this select 4], _file] remoteExec ["execVM",0,true]; 
+		private _enableFlareEnhance = getMissionConfigValue ["ACTIVAR_BENGALAS_MEJORADAS",  0] == 1; 
+		if (_enableFlareEnhance) then {
+			deleteVehicle (_this select 6);
+		};
```
Comment (IgnacioKase) [inline core/scripts/flares/activate_mortar.sqf:19]: minor but you could name the variables in `_this`

File: core/scripts/flares/activate_mortar.sqf [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+                          Realizado por MIV
+*******************************************************************************/
+
+params ["_mortar"];
+
+_mortar setVehicleAmmo 0.0; 
+{
+	_mortar addmagazine "8Rnd_82mm_Mo_Flare_white"
```
Comment (IgnacioKase) [inline core/scripts/flares/activate_mortar.sqf]: not really good indentation here! (and bellow actually)

**Review by IgnacioKase — APPROVED** [review]
LGTM! But some minor things

File: core/scripts/flares/activate_mortar.sqf (hunk 3 above)
Comment (VulturARG) [reply core/scripts/flares/activate_mortar.sqf]: The indentation is set by the VSC. They are two tabs. It's just a matter of how it looks

File: core/scripts/flares/activate_mortar.sqf (hunk 1 above)
Comment (VulturARG) [reply core/scripts/flares/activate_mortar.sqf]: Done

**Pull request merged**